src/DigitalProductPassport/Battery/ManufacturingDataSheet_v0.1.py

```diff
@@ -0,0 +1,377 @@
+    manufacturer_information: ManufacturerInformation = Field(
+        ...,
+        title="Manufacturer Information",
+        description="The details of the battery manufacturer",
+    )
+    manufacturing_location: ManufacturingLocation = Field(
+        ...,
+        title="Manufacturing Location",
+        description="The location details of the battery manufacturing plant",
+    )
+    manufacturing_date: str = Field(
+        ...,
+        title="Manufacturing Date",
+        description="The date of manufacture using month and year",
+        pattern=r"^\d{4}-(0[1-9]|1[0-2])$",
+        examples=["2023-07"],
+    )
+    weight: float = Field(
+        ...,
+        title="Weight",
+        description="The total net weight of the product in kilograms (kg)",
+        examples=[450.0],
+    )
+    capacity: float = Field(
+        ...,
+        title="Capacity",
+        description="The total number of ampere-hours (Ah) that can be withdrawn from a fully charged battery under reference conditions",
+        examples=[100.0],
+    )
+    power: float = Field(
+        ...,
+        title="Power",
+        description="The original power capability of the battery in Watts",
+        examples=[25000.0],
+    )
+    cell_type: str = Field(
+        ...,
+        max_length=250,
+        title="Cell Type",
+        description="The type of cells used in the battery pack",
+        examples=["sodium-ion"],
+    )
+    resistance: float = Field(
+        ...,
+        title="Resistance",
+        description="The internal resistance of the battery pack",
+        examples=[0],
+    )
+    round_trip_efficiency: Optional[RoundTripEfficiency] = Field(
+        None,
+        title="Round Trip Efficiency",
+        description="The details of the round trip energy efficiency in energy storages",
+    )
+    voltage_levels: VoltageLevels = Field(
+        ...,
+        title="Voltage Levels",
+        description="The details of the voltage levels of the battery",
+    )
+    temperature_range: TemperatureRange = Field(
+        ...,
+        title="Temperature Range",
+        description="The details of the acceptable temperature values of the battery",
+    )
+    expected_lifetime: ExpectedLifetime = Field(
+        ...,
+        title="Expected Lifetime",
+        description="The details of the battery lifetime",
+    )
+    material_composition: MaterialComposition = Field(
+        ...,
+        title="Material Composition",
+        description="The details of the material composition of the battery",
+    )
+    recycled_content: List[RecycledContent] = Field(
+        ...,
+        title="Recycled Content",
+        description="The recycled content information present in the battery",
+    )
+    renewable_content: List[RenewableContent] = Field(
+        ...,
+        title="Renewable Content",
+        description="The renewable content information present in the battery",
+    )
+    legal_conformity: LegalConformity = Field(
```

Comment:
`extinguishing_agent` was not supposed to be removed altogether, just changed to a list of strings.